src/style.css

```
@@ -1 +1,64 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
```

common font property here is only font-family, because you will have other typography variants in the next tasks
```suggestion
  font-family: "Roboto", sans-serif;
```